Makefile

```diff
@@ -76,6 +76,22 @@ install: LDFLAGS += -X "github.com/cockroachdb/cockroach/util.buildDeps=$(shell
 install:
 	$(GO) install -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v
 
+# Build, but do not run the tests.
+# PKG is expanded and all packages are built and moved to their directory.
+# If STATIC=1, tests are statically linked.
+# eg: to statically build the sql tests, run:
+#   make testbuild PKG=./sql STATIC=1
+.PHONY: testbuild
+testbuild: TESTS := $(shell $(GO) list $(PKG))
+testbuild: GOFLAGS += -c
+testbuild:
+	for p in $(TESTS); do \
+	  NAME=$$(basename "$$p"); \
+	  OUT="$$NAME.test"; \
+	  DIR=$$($(GO) list -f {{.Dir}} ./...$$NAME); \
+	  $(GO) test $(GOFLAGS) -o "$$DIR"/"$$OUT" -ldflags '$(LDFLAGS)' "$$p" $(TESTFLAGS) || break; \
+	done
```

Comment:
We had a similar rule before. See 3d44a2767d2c79ece0553d3b8ff0c2988d7f8d7f where it was removed. I think you need to add `-tags '$(TAGS)'` as well.